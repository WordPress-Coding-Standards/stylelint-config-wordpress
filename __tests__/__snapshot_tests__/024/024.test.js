'use strict';

const stylelint = require( 'stylelint' ),
	systemTestUtils = require( '../systemTestUtils.js' );

it( '024', () => {
	return stylelint.lint({
		files: [systemTestUtils.caseStylesheetGlob( '024' )],
		configFile: systemTestUtils.caseConfig( '024' ),
	}).then( ({ results }) => {
		expect( systemTestUtils.prepResults( results ) ).toMatchSnapshot();
	});
});
