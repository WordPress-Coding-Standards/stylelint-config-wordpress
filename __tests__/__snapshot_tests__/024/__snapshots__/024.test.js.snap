// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`024 1`] = `
Array [
  Object {
    "deprecations": Array [],
    "errored": true,
    "ignored": undefined,
    "invalidOptionWarnings": Array [],
    "parseErrors": Array [],
    "source": "024/stylesheet.css",
    "warnings": Array [
      Object {
        "column": 1,
        "line": 58,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 103,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 406,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 629,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 634,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 804,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 809,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 814,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 1011,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 1,
        "line": 1478,
        "rule": "comment-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before comment (comment-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 78,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 115,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 121,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 146,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 149,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 159,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 180,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 185,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 188,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 195,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 199,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 203,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 206,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 213,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 218,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 222,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 225,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 230,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 233,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 238,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 244,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 248,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 251,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 254,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 344,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 348,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 351,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 356,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 359,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 367,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 371,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 375,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 381,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 384,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 391,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 434,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 438,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 441,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 444,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 448,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 456,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 459,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 519,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 523,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 527,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 530,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 541,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 558,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 582,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 586,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 591,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 596,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 646,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 682,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 686,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 690,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 693,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 697,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 700,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 705,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 708,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 711,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 737,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 740,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 743,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 768,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 774,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 784,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 790,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 794,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 876,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 921,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 936,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 940,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 946,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 957,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 963,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 966,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 970,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 988,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 994,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 997,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1000,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1104,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1117,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1159,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1335,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1441,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1446,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1454,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 2,
        "line": 1457,
        "rule": "rule-empty-line-before",
        "severity": "error",
        "text": "Expected empty line before rule (rule-empty-line-before)",
      },
      Object {
        "column": 37,
        "line": 222,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 64,
        "line": 251,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 73,
        "line": 620,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 52,
        "line": 1226,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 30,
        "line": 1306,
        "rule": "selector-list-comma-newline-after",
        "severity": "error",
        "text": "Expected newline after \\",\\" (selector-list-comma-newline-after)",
      },
      Object {
        "column": 3,
        "line": 771,
        "rule": "shorthand-property-no-redundant-values",
        "severity": "error",
        "text": "Unexpected longhand value '0 0 5px 0' instead of '0 0 5px' (shorthand-property-no-redundant-values)",
      },
    ],
  },
]
`;
